Tella/Scenes/Resources/Views/ResourcePDFView.swift

```diff
@@ -11,16 +11,18 @@ import SwiftUI
 struct ResourcePDFView: View {
     var file: URL
     var resourceTitle: String
+    @State private var navigationBarHidden = false
     var body: some View {
         ZStack {
-            QuickLookView(file: file)
+            PDFKitView(url: file)
```

Comment:
@dhekra-rouatbi here I added PDFKit because it provides a more flexible approach for customizing the UI. In this case we wanted to hide the navigation bar when the user scrolls down and show it again when it scrolls up. However I wanted to double check if you think it's fine to use it here. I couldn't find a solution using QuickLook for this kind of customization

Reply:
honestly i don't have idea about `PDFKit` but if it  provides more flexible approach for customizing the UI, let's try it